src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -95,129 +162,977 @@ const ActorInit En_Skb_InitVars = {
+                        Audio_PlayActorSound2(&this->actor, NA_SE_EN_STALKID_DAMAGE);
+                        this->unk_3D2 = 0x50;
+                    } else {
+                        this->unk_3D2 = 3;
+                    }
+                    this->unk_3E6 = 0xB;
+                    this->unk_22C = 1.0f;
+                    this->unk_230 = 0.5f;
+                    func_800BCB70(&this->actor, 0x4000, 0xFF, 0, 8);
+                    func_809962D4(this);
+                    break;
+
+                case 4:
+                    this->unk_3D2 = 0x28;
+                    this->unk_3E6 = 0x14;
+                    this->unk_22C = 1.0f;
+                    this->unk_230 = 0.5f;
+                    func_800BCB70(&this->actor, 0x4000, 0xFF, 0, 8);
+                    Audio_PlayActorSound2(&this->actor, NA_SE_EN_STALKID_DAMAGE);
+                    func_800BDC5C(&this->skelAnime, sAnimations, 3);
+                    func_809963C4(this);
+                    break;
+
+                case 12:
+                case 14:
+                    this->unk_3D8 |= 1;
+
+                case 15:
+                    if ((player->swordAnimation == 4) || (player->swordAnimation == 11) ||
+                        (player->swordAnimation == 22) || (player->swordAnimation == 23)) {
+                        this->unk_3D8 |= 1;
+                    }
+
+                case 13:
+                    func_800BCB70(&this->actor, 0x4000, 0xFF, 0, 8);
+                    func_800BDC5C(&this->skelAnime, sAnimations, 3);
+                    func_80995F98(this);
+                    break;
+            }
+        }
+    }
+}
+
+void func_80996AD0(EnSkb* this, GlobalContext* globalCtx) {
+    if (1) {};
+
+    if ((this->actionFunc != func_80996284) && (this->unk_3E2 != 1)) {
+        if (this->unk_3E4 != 0) {
+            CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+
+        if (((this->unk_3DE != 0) || (this->unk_3D0 >= 11)) && (this->unk_3DE != 1) && (this->unk_3DE != 4) &&
+            (this->unk_3DE != 6) && (this->unk_3DE != 7) &&
+            ((this->actor.colorFilterTimer == 0) || !(this->actor.colorFilterParams & 0x4000))) {
+            CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+
+        if ((this->actionFunc != func_80996284) && (this->actionFunc != func_809964A0)) {
+            CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+    }
+}
+
+void func_80996BEC(EnSkb* this, GlobalContext* globalCtx) {
+    static Color_RGBA8 D_80997550 = { 170, 255, 255, 255 };
+    static Color_RGBA8 D_80997554 = { 200, 200, 255, 255 };
+    static Vec3f D_80997558 = { 0.0f, -1.0, 0.0f };
+    Vec3f sp84;
+    s32 i;
+    s16 yaw;
+    s32 end;
+
+    if (this->unk_3D8 & 2) {
+        end = 13;
+    } else {
+        end = 14;
+    }
+
+    func_800F0568(globalCtx, &this->actor.world.pos, 30, 0x28CB);
```

Comment:
```suggestion
    func_800F0568(globalCtx, &this->actor.world.pos, 30, NA_SE_EV_ICE_BROKEN);
```
Same comment as above, may have to change names